pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -225,6 +254,16 @@ type ClusterAffinity struct {
 	ExcludeClusters []string `json:"exclude,omitempty"`
 }
 
+// OrderedClusterAffinity represents a group of cluster as well as an order that
+// Karmada scheduler should inspect with.
+type OrderedClusterAffinity struct {
+	ClusterAffinity `json:",inline"`
+
+	// Order of the cluster group that Karmada scheduler should inspect with, in the range 1-100.
+	// +required
```

Comment:
> in the range 1-100.

Why define a range?

```suggestion
	// +required
	// +kubebuilder:validation:Minimum=1
	// +kubebuilder:validation:Maximum=100
```

Reply:
It is same as https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity-weight, and It should be sufficient in a cluster group.

Reply:
Yes, exactly. Since we probably record each group's scheduling results, the range can not be too wide.